Cargo.toml

```
@@ -13,7 +13,7 @@ repository = "https://github.com/Synphonyte/leptos-struct-table"
 
 [dependencies]
 leptos = { version = "0.6", features = ["nightly"] }
-leptos-struct-table-macro = { version = "0.10.0" }
+leptos-struct-table-macro = { version = "0.10.0", path="../leptos-struct-table-macro" }
```

This obviously needs to change, most likely to 

leptos-struct-table-macro = { version = "0.11.0" }
```suggestion
leptos-struct-table-macro = { version = "0.11.0" }
```

---

Absolutely correct. I'll do that locally.